{
  "_args": [
    [
      "assert-error@1.0.3",
      "F:\\Learn\\simple_desktop"
    ]
  ],
  "_from": "assert-error@1.0.3",
  "_id": "assert-error@1.0.3",
  "_inBundle": false,
  "_integrity": "sha1-ReesJZtc7gOVOthHnEeNm0zLNBs=",
  "_location": "/assert-error",
  "_phantomChildren": {},
  "_requested": {
    "type": "version",
    "registry": true,
    "raw": "assert-error@1.0.3",
    "name": "assert-error",
    "escapedName": "assert-error",
    "rawSpec": "1.0.3",
    "saveSpec": null,
    "fetchSpec": "1.0.3"
  },
  "_requiredBy": [
    "/runtimejs"
  ],
  "_resolved": "https://registry.npm.taobao.org/assert-error/download/assert-error-1.0.3.tgz",
  "_spec": "1.0.3",
  "_where": "F:\\Learn\\simple_desktop",
  "author": {
    "name": "Sergii Iefremov",
    "email": "iefserge@runtimejs.org"
  },
  "bugs": {
    "url": "https://github.com/iefserge/assert-error/issues"
  },
  "description": "Basic assert function that throws provided error object",
  "devDependencies": {
    "tape": "^4.0.0"
  },
  "homepage": "https://github.com/iefserge/assert-error",
  "keywords": [
    "assert",
    "error"
  ],
  "license": "Apache-2.0",
  "main": "index.js",
  "name": "assert-error",
  "repository": {
    "type": "git",
    "url": "git+ssh://git@github.com/iefserge/assert-error.git"
  },
  "scripts": {
    "test": "tape test.js"
  },
  "version": "1.0.3"
}
